src/database/CMakeLists.txt

```diff
@@ -1,5 +1,29 @@
-add_library(example-lib Example.cpp)
+cmake_minimum_required(VERSION 3.12)
 
-target_include_directories(example-lib PRIVATE
-    ${INCLUDE_DIR}
+add_library(database-lib database.cpp)
+add_executable(database-exec database.cpp)
+
+find_package(libmongocxx REQUIRED)
+find_package(mongocxx REQUIRED)
+
+message("LIBBSONCXX_INCLUDE_DIRS=${LIBBSONCXX_INCLUDE_DIRS}")
+message("LIBBSONCXX_LIBRARIES=${LIBBSONCXX_LIBRARIES}")
+
+message("LIBMONGOCXX_INCLUDE_DIRS=${LIBMONGOCXX_INCLUDE_DIRS}")
+message("LIBMONGOCXX_LIBRARIES=${LIBMONGOCXX_LIBRARIES}")
+
+
+target_include_directories(database-exec PUBLIC 
```

Comment:
Please add only directories to include to `target_include_directories` not libraries path